llms/README.md

```diff
@@ -46,7 +46,7 @@ You can convert models in the Python API with:
 ```python
 from mlx_lm import convert
 
-upload_repo = "mistralai/Mistral-7B-Instruct-v0.1"
+upload_repo = "mlx-community/My-Mistral-7B-v0.1-4bit"
```

Comment:
Thank you for adding the model. 
- Why is this switching to `mlx-community/My-Mistral-7B-v0.1-4bit`? 
- Is this necessary for StarCoder2 model implementation?

Reply:
Hi this is just typo from elsewhere

Reply:
Ok. In that case, please switch it back to what it was earlier.

Reply:
@sugatoray I think what @Muhtasham meant is that the previous version is a typo (which it is). It doesn't make sense to point to the mistralai repo as an example upload name. We should have something more suggestive of a custom repo. I would use the same thing we use later in the `README.md`:

```
mlx-community/my-4bit-mistral
```

As for including it in the PR, I think it is fine (or you can send another if you prefer).